packages/core/src/local-admin-api.ts

```diff
@@ -0,0 +1,37 @@
+import { AdminApi, DiagnosticsLogger, LogStyle } from '@ceramicnetwork/common'
+import { StreamID } from '@ceramicnetwork/streamid'
+import { LocalIndexApi } from './indexing/local-index-api.js'
+
+/**
+ * AdminApi for Ceramic core.
+ */
+export class LocalAdminApi implements AdminApi {
+  constructor(
+    private readonly indexApi: LocalIndexApi,
+    private readonly logger: DiagnosticsLogger
+  ) {}
+
+  async startIndexingModels(modelsIDs: Array<StreamID>): Promise<void> {
+    this.logger.log(LogStyle.info, `Local Admin Api will add model ids to index ${modelsIDs}`)
+    await this.indexApi.indexModels(modelsIDs)
+    this.logger.log(LogStyle.info, `Local Admin Api did add model ids to index ${modelsIDs}`)
```

Comment:
logging both before and after the operation feels like overkill to me.  I'd suggest doing it before only